jekyll/_cci2/orb-concepts.md

```diff
@@ -7,203 +7,171 @@ categories: [getting-started]
 order: 1
 ---
 
-CircleCI orbs are shareable packages of configuration elements, including jobs, commands, and executors. CircleCI provides certified orbs, along with 3rd-party orbs authored by CircleCI partners. It is best practice to first evaluate whether any of these existing orbs will help you in your configuration workflow. Refer to the CircleCI Orbs Registry for the complete list of certified orbs.
+* TOC
+{:toc}
 
-Before using orbs, you should first familiarize yourself with some basic core concepts of orbs and how they are structured and operate. Gaining a basic understanding of these core concepts will enable you to leverage orbs and use them easily in your own environments.
+## Quick Start
 
-### Certified vs. 3rd-Party Orbs
+[CircleCI orbs](https://circleci.com/orbs/) are shareable packages of configuration elements, including [jobs]({{site.baseurl}}/2.0/reusing-config/#authoring-parameterized-jobs), [commands]({{site.baseurl}}/2.0/reusing-config/#authoring-reusable-commands), and [executors]({{site.baseurl}}/2.0/reusing-config/#authoring-reusable-executors). Orbs make writing and customizing CircleCI config simple. The reusable configuration elements used in orbs are explained fully in the [Reusable Configuration Reference]({{site.baseurl}}/2.0/reusing-config/).
 
-CircleCI has available a number of individual orbs that have been tested and certified to work with the platform. These orbs will be treated as part of the platform; all other orbs are considered 3rd-party orbs. Note: The Admin of your org must opt-in to 3rd-party uncertified orb usage on the Settings > Security page for your org.
+## Reusable Configuration
 
-All orbs are open, meaning that anyone can use them and see their source.
+CircleCI's [Reusable Configuration]({{site.baseurl}}/2.0/reusing-config/) features allow you to define parametrizable configuration elements and re-use those elements throughout a project config file. It is recommended you become familiar with the full [Configuration Reference]({{site.baseurl}}/2.0/configuration-reference/) features before moving on to the [Reusable Configuration Reference]({{site.baseurl}}/2.0/reusing-config/).
 
-### Design Methodology
+### Commands
+Commands contain one or more [steps]() in which [parameters]() can be used to modify behavior. Commands are the logic of orbs and are responsible for executing steps such as [checking out code](https://circleci.com/docs/2.0/configuration-reference/#checkout), or running shell code, for example, running BASH or running CLI tools. For more information see the [Authoring Reusable Commands]({{site.baseurl}}/2.0/reusing-config/#authoring-reusable-commands) guide.
```

Comment:
```
Commands are the logic of orbs and are responsible for executing steps such as checking out code, or running shell code, for example, running BASH or running CLI tools
|
V
Commands are the executable portions of orbs, definining steps such as checking out code, running shell scripts, installing CLI tools, or any other commands that should run.
```

`Commands are the logic of orbs` doesn't sit quite right with me, since we also have `condition`al stanzas in the YAML itself. 

`responsible for executing steps such as checking out code, or running shell code, for example, running BASH or running CLI tools` feels a touch awkward, rephrasing it or making it more concise would help.

Reply:
I am slightly concerned with using the word "execution" as well because I do not want to confuse folks on Executors. Thinking on this one for the moment.